cpp/open3d/t/geometry/TriangleMesh.h

```diff
@@ -594,6 +594,18 @@ class TriangleMesh : public Geometry, public DrawableGeometry {
             core::Dtype int_dtype = core::Int64,
             const core::Device &device = core::Device("CPU:0"));
 
+    /// Create a mesh from a scalar volume by computing the isocontour.
+    /// This method uses the Flying Edges dual contouring method that similarly
+    /// like Marching Cubes computes the isocontour.
+    /// \param volume 3D tensor with the volume.
+    /// \param contour_values A list of contour values at which isocontours will
+    /// be generated.The values used for contouring. The default \param device
+    /// The device for the returned mesh.
+    static TriangleMesh CreateFromVolume(
```

Comment:
Not sure if `CreateFromVolume` is easily understood. How about using `CreateIsoSurfaces` instead? The VTK blog and paper refers to isocontours and 3D image instead of volume, while wikipedia article is about isosurfaces.  Matlab and other sources seem to prefer iso surfaces. 3D scalar field is perhaps more clear than volume.

Reply:
changed function name to CreateIsosurfaces/create_isosurfaces